docs/installer-live-iso.md

```diff
@@ -43,7 +43,7 @@ The base live ISO is extracted from a container image. Run the container
 image containing the ISO, copy the ISO, and then stop and remove the container. 
 
 ````
-podman run -dt --name livecdsrc quay.io/ocpmetal/livecd-iso:rhcos-livecd
+podman run -dt --name livecdsrc quay.io/ocpmetal/livecd-iso:v4.6
```

Comment:
is quay.io/ocpmetal/livecd-iso:v4.6 going away? Seems like the source should be from https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/ as per https://github.com/openshift/assisted-service/pull/358.